catalog/config-schema.json

```
@@ -96,6 +96,10 @@
     "qurator": {
       "type": "boolean",
       "description": "Enable Qurator AI Assistant (powered by Amazon Bedrock)"
+    },
+    "stack_version": {
+      "type": "string",
+      "description": "Stack release version, uses semver"
```

If it is not semver, then I'd not mention it then

```suggestion
      "description": "Stack release version"
```

---

ok. it's not a strict SemVer, and it also may have a suffix in non-release (dev / nightly) versions

---

Aha. I'm just saying `semver**-ish**` makes more questions than gives answers. If there is a short explanation of the format (semver + these diverges), I'd add it here

---

nah, let's just omit it